package/nginx-http.conf

```diff
@@ -20,5 +20,9 @@ server {
         return 301 https://$host$request_uri;
     }
 
+    location /suma {
```

Comment:
Is the data required in plain `http`? I thought SM is also consuming other endpoints which are all only available via `https`?

Reply:
It is a request that RMT works more with http and that https is just the better workflow.

See: https://trello.com/c/fGuUG7O1/3105-rmt-allow-registration-over-http

Reply:
Okay didn't know about this problem. Than I'm fine with this (not with with autoyast being unable to import certs...)